tensorflow_serving/model_servers/main.cc

```diff
@@ -352,6 +354,22 @@ tensorflow::serving::PlatformConfigMap ParsePlatformConfigMap(
   return platform_config_map;
 }
 
+tensorflow::Status readkey( const tensorflow::string& filename, tensorflow::string& data )
```

Comment:
Please include a brief comment before every new function describing what it does.